visualizations/frontend/filters/v2/TRangeInput.vue

```diff
@@ -76,14 +77,20 @@
         },
         methods: {
             update(e) {
-                if (e.minValue === e.min && e.maxValue == e.max) {
-                    this.unsetFilterValue('min', true);
-                    this.unsetFilterValue('max', true);
-                } else {
-                    this.barMinValue = e.minValue;
-                    this.barMaxValue = e.maxValue;
+                if (this.throttleId) {
+                    clearInterval(this.throttleId);
```

Comment:
I read this about using `clearInterval` with a timeoutId

"It's worth noting that the pool of IDs used by [setInterval()](https://developer.mozilla.org/en-US/docs/Web/API/setInterval) and [setTimeout()](https://developer.mozilla.org/en-US/docs/Web/API/setTimeout) are shared, which means you can technically use clearInterval() and [clearTimeout()](https://developer.mozilla.org/en-US/docs/Web/API/clearTimeout) interchangeably. However, for clarity, you should avoid doing so." - https://developer.mozilla.org/en-US/docs/Web/API/clearInterval

Maybe if you could use `clearTimeout` instead. However, I don't think you need it, unless you need to cancel the timeout for some reason.

Reply:
You want to cancel the timeout if some event is fire (basically this is the throttling behaviour we are looking for).
If you call `update`, and then you call it again before 250ms has happened, you will cancel the first setTimeout (because you do not want to execute its callback) and you will set another setTimeout...
You will keep doing this until no event is calling update within this window of 250ms.

Reply:
ah! Of course, makes sense.

Reply:
Oh thank you @sareg0 for finding my `clearInterval` mistake 😛